x-pack/elastic-agent/docs/upgrade-elastic-agent.asciidoc

```diff
@@ -0,0 +1,30 @@
+[[upgrade-elastic-agent]]
+[role="xpack"]
+= Upgrade {agent}
+
+beta[]
+
+NOTE: This process is not supported for upgrading DEB/RPM packages or Docker
+images.
+
+To upgrade {agent} to a new version:
```

Comment:
I need to add a section here to describe how to do this from fleet. I need help, though, because I can't actually run through the steps.

Reply:
I will do a follow with instruction for that.

Reply:
added as a to-do